src/main/java/com/rapid7/client/dcerpc/dto/ContextHandle.java

```diff
@@ -76,7 +76,7 @@ public static ContextHandle fromHex(final String hString) {
         if (hString == null || hString.length() > (bytes.length * 2)) {
             throw new IllegalArgumentException("hString is invalid: " + hString);
         }
-        final byte[] handle = Hex.decode(hString);
+        final byte[] handle = BaseEncoding.base16().decode(hString.toUpperCase());
```

Comment:
Does it require being upper case to decode then? As we are not obviously uppercasing in the other usages

Reply:
The BaseEncoding encodes and decodes using uppercase characters by default.